// jest-without-globals/dist/jest-without-globals.cjs.development.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var jestWG = jest;
var expectWG = expect; // Jest's list of globals: https://jestjs.io/docs/en/api

var afterAllWG = afterAll;
var afterEachWG = afterEach;
var beforeAllWG = beforeAll;
var beforeEachWG = beforeEach;
var describeWG = describe;
var fdescribeWG = fdescribe; // alias for describe.only

var xdescribeWG = xdescribe; // alias for describe.skip

var testWG = test;
var itWG = it; // alias for test

var fitWG = fit; // alias for it.only

var xtestWG = xtest; // alias for test.skip

var xitWG = xit; // alias for it.skip
 // alias for it.skip

exports.afterAll = afterAllWG;
exports.afterEach = afterEachWG;
exports.beforeAll = beforeAllWG;
exports.beforeEach = beforeEachWG;
exports.describe = describeWG;
exports.expect = expectWG;
exports.fdescribe = fdescribeWG;
exports.fit = fitWG;
exports.it = itWG;
exports.jest = jestWG;
exports.test = testWG;
exports.xdescribe = xdescribeWG;
exports.xit = xitWG;
exports.xtest = xtestWG;
